VoxelSniperCore/src/main/java/com/github/kevindagame/command/VoxelBrushCommand.java

```diff
@@ -59,7 +59,7 @@ public boolean doCommand(IPlayer player, final String[] args) {
             int originalSize = snipeData.getBrushSize();
             snipeData.setBrushSize(Integer.parseInt(args[0]));
 
-            new PlayerBrushSizeChangedEvent(player, currentToolId, originalSize, snipeData.getBrushSize()).callEvent();
+            new PlayerBrushSizeChangedEvent(player, currentToolId, sniper.getBrush(currentToolId), originalSize, snipeData.getBrushSize()).callEvent();
```

Comment:
This event is now Cancellable, right? that means we shouldn't change the brushsize when the event is cancelled. (I think this also applies to the other events that are now Cancellable.